### PR TITLE
Merge branch 'feature-develop' into 'develop

### PR DESCRIPTION
**Pull request by zacpez**

### CODE REVIEW

File: cit-api/pipeline/views/opportunity.py [hunk 1]
```diff
@@ -42,14 +57,98 @@ def get_queryset(self):
...
+        zoning = self.request.query_params.get('zoning', None)
+        if(zoning is not None):
+            zonings = zoning.split(',')
+            queryset = queryset.filter(Q(land_use_zoning__in=zonings) | Q(ocp_zoning_code__in=zonings))
+
+        return queryset
+
+    def service_queryset(self, queryset, exclude_unknowns, service_name):
+        service_connected = self.request.query_params.get(service_name, None)
+        print("----------")
```
Comment (HypnoMama) [inline cit-api/pipeline/views/opportunity.py:92]: Do we want to get rid of these print statements before merging?

**Review by HypnoMama — APPROVED** [review]

**Review by BrendanBeachBC — COMMENTED** [review]
One question about one of the data migrations, and a couple of other minor things. Otherwise looks good once the migration question has been answered

File: cit-api/pipeline/views/opportunity.py (hunk 1 above)
Comment (BrendanBeachBC) [reply cit-api/pipeline/views/opportunity.py:92]: I agree that we should remove these, but I'll be taking them out in an upcoming PR and I think they're fine to be in develop until then.

Comment (HypnoMama) [reply cit-api/pipeline/views/opportunity.py:92]: Okay sounds good!

File: cit-api/pipeline/migrations/0136_auto_20210323_1547.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.16 on 2021-03-23 15:47
+
+from django.db import migrations, models
+
+def populate_statuses(apps, schema_editor):
+    LandUseZoning = apps.get_model("pipeline", "LandUseZoning")
+    LandUseZoning.objects.get(code="INDH").delete();
+    zonings = [
+        ["Industrial (Heavy)", "", "INDH"]
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit-api/pipeline/migrations/0136_auto_20210323_1547.py:9]: Yes, we don't have a description yet.

[the remark this answers is not in the document]
Comment (zacpez) [reply cit-api/pipeline/migrations/0136_auto_20210323_1547.py:9]: Nor is it shown anywhere, yet. Was to be a tooltip somewhere I believe.

File: cit-api/pipeline/serializers/opportunity.py [hunk 3]
```diff
@@ -47,5 +183,210 @@ class Meta:
...
+            # TODO: validation filtered_first_nations = IndianReserveBandName.objects.filter(name__in=first_nations)
+
+            # create distances
+
+            for nation in first_nations:
+                # create and append
+                filtered_first_nations_distance = IndianReserveBandDistance.objects.create(
+                    **nation)
+                filtered_first_nations_distances.append(
+                    filtered_first_nations_distance.pk)
```
[the remark this answers is not in the document]
Comment (zacpez) [reply cit-api/pipeline/serializers/opportunity.py:361]: Ooh yeah, that was there as conversation visualization during a screen share.

File: cit-api/pipeline/migrations/0136_auto_20210323_1547.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (BrendanBeachBC) [reply cit-api/pipeline/migrations/0136_auto_20210323_1547.py:9]: Okay cool.

**Review by BrendanBeachBC — APPROVED** [review]

**Pull request merged**